tests/flycheck-hledger-test.el

```diff
@@ -71,22 +71,42 @@ The error message.
     :expected-line "1"
     :expected-column "10"
     :expected-end-column "20"
-    :expected-message "\nThe error message.\n"
+    :expected-message "The error message.\n"
     :output "hledger: Error: ./file.ledger:1:10-20:
 3 | 2022-01-01
   |     a               1
 
 The error message.
 "))
 
+(defconst flycheck-hledger-test-balance-error-windows
+  '(
+    :expected-file "./file.ledger"
+    :expected-line "2"
+    :expected-message "Strict account checking is enabled, and\naccount \"a\" has not been declared.\nConsider adding an account directive. Examples:\n\naccount a\naccount a    ; type:A  ; (L,E,R,X,C,V)\n\n"
```

Comment:
could this message be as simple as the ones of other error objects? This error object is only meant to test the behavior on Windows. Feel free to add another error object for a different issue you are fixing (which should probably go in a separate and atomic commit)

Reply:
To clarify: since the other tests don’t use real messages, should I take a real error message from hledger and simplify it to match?

Reply:
What I meant was to make as many error objects as bugs you are fixing instead of checking all bug fixes with just one error object.